### ISSUE TITLE
Favorite home

### ISSUE THREAD

**Issue by DamyanBG** [DamyanBG/real-estate-react-app#380]
Add ability for users to favorite a home.

**Comment by DamyanBG** [comment]
@tihomirtx88 You can take this task

**Comment by DamyanBG** [comment]
On home details page there is a button - Add to Favorites. Create post request to the back end on route /favorite-home with JSON which includes the current user id + the current home id. When you receive successful  response, change the text to Favorited on the button and disable it.

**Comment by khanSoliheen** [comment]
@DamyanBG , I tried to run this app. When I am hitting an api from main branch. I do not get any response from backend.https://real-estate-flask-rest-api.onrender.com/user/register-user or signin

**Comment by DamyanBG** [comment]
Because I am not paying anymore for the subscription of the database. Do you want to contribute to this project? If yes, I am building version 2 of it.

**Comment by khanSoliheen** [comment]
@DamyanBG , Thank you for considering me for your project! I would be delighted to contribute to version 2 and help bring your vision to life. Let me know how I can get involved and what the next steps are. Looking forward to working together!

**Comment by DamyanBG** [comment]
@khanSoliheen 

https://github.com/DamyanBG/real-estate-nextjs-v2

Here is the new repo, these days I will commit some code. After that I will add README and some tasks. Are you familiar with Docker?

**Comment by DamyanBG** [comment]
@khanSoliheen  this can be your first issue:

https://github.com/DamyanBG/crypto-insights/issues/1

It is not related to Real Estate, but I think it is good.

**Comment by khanSoliheen** [comment]
@DamyanBG , Yes, I'm familiar with Docker as well. Let me know if you need any help or have specific requirements related to Docker in the project.

**Comment by khanSoliheen** [comment]
Thanks for sharing the new issue. I’ll review it and update you on the coloring aspect.
